### DISCUSSION TITLE
[Feature Request] Show nicely formatted datetime values in View Results instead of raw timestamp values (Clickhouse)

### DISCUSSION THREAD

**Discussion by david-zh-cn** [apache/superset#19180] [Ideas]
![image](https://user-images.githubusercontent.com/44357168/126071849-f2590623-8228-45b2-9830-4a2f0f7437f7.png)

前端对clickhouse的datetime64的支持，能够支持毫秒级别

**Comment by srinify** [comment]
@junlincc I think this is a feature request to improve how the values in the results preview are shown, so they're more consistent.

**Comment by david-zh-cn** [comment]
@srinify you are right. it is very useful when we want to use superset in log domain, because It is very common to log in milliseconds.so thank for you support.
